src/REPLMode.jl

```diff
@@ -1203,6 +1238,41 @@ is modified.
     """,
 ),
 ], #package
+
+["registry"] => CommandDeclaration[
+(
+    CMD_REGISTRY_ADD,
+    ["add"],
+    do_registry_add!,
+    (ARG_RAW, []),
+    [],
+    nothing,
+),
+(
+    CMD_REGISTRY_RM,
+    ["rm"],
```

Comment:
it may be a good idea to include a `remove` alias to mirror the `package` REPL API?

Reply:
Yes, didn't actually know about that alias!